packages/ilios-common/addon/components/editable-field.hbs

```diff
@@ -70,18 +74,20 @@
                   />
                 {{/if}}
               </button>
-              {{#if isTruncated}}
-                <EllipsisIcon />
-                <button
-                  class="expand-text-button"
-                  type="button"
-                  aria-label={{t "general.expand"}}
-                  title={{t "general.expand"}}
-                  data-test-expand
-                  {{on "click" expand}}
-                >
-                  <FaIcon @icon="angles-down" />
-                </button>
+              {{#if isFaded}}
+                <div class="fade-text-control">
+                  <EllipsisIcon />
```

Comment:
I think we should ditch the ellipsis, it doesn't make as much sense in a faded world.

Reply:
Sounds like a good discussion item for next meeting.

Reply:
Actually, now that I think about it more, ditching this makes perfect sense, since we're not actually chopping off any text, which is usually what the ellipses represents. Will ditch.